File: app/src/main/java/com/app/addviu/model/CommonSuccess.java
package com.app.addviu.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CommonSuccess {

@SerializedName("status")
@Expose
public int status;
@SerializedName("message")
@Expose
public String message;

}

File: app/src/main/java/com/app/addviu/model/CommonSuccess.kt
package com.app.addviu.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CommonSuccess {
    @SerializedName("status")
    @Expose
    var status = 0

    @SerializedName("message")
    @Expose
    var message: String = ""
}